eva/binder/statement_binder.py

```diff
@@ -207,6 +211,10 @@ def _bind_tuple_expr(self, node: TupleValueExpression):
         table_alias, col_obj = self._binder_context.get_binded_column(
             node.col_name, node.table_alias
         )
+        if node.col_name == ColumnName.audio:
```

Comment:
Do we need it for deletion? We will just drop the video in both cases, right?

Reply:
I don't follow :sweat_smile: I added this in  `_bind_tuple_expr` but from the UI it makes it look like I added this in the bind delete part. Is that what is causing the confusion?

Reply:
I see. My bad

Reply:
I see. My bad